mkosi.profiles/desktop/mkosi.extra/usr/lib/tmpfiles.d/10-particleos-desktop.conf

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+L /etc/dbus-1
```

Comment:
Isn't this something to fix in Fedora? dbus looks in /usr as well no?

Reply:
So I added this because otherwise `rtkit-daemon` (pulled in by pipewire) fails to start on boot, because `/etc/dbus-1/system.d/org.freedesktop.RealtimeKit1.conf`. I suppose we should fix that rpm to put that in `/usr` instead?

Reply:
Yeah, this is explicitly installed as a config file: https://src.fedoraproject.org/rpms/rtkit/blob/rawhide/f/rtkit.spec#_77

Reply:
@davide125 Can you file a PR to move it? I think @keszybz maintains rtkit in Fedora

Reply:
https://src.fedoraproject.org/rpms/rtkit/pull-request/3

Reply:
I think we can drop this now